recipes-kernel/linux/linux-nezha-dev.bb

```diff
@@ -0,0 +1,22 @@
+require recipes-kernel/linux/linux-mainline-common.inc
+FILESEXTRAPATHS:prepend = "${FILE_DIRNAME}/linux-nezha:"
+SUMMARY = "Nezha dev kernel recipe"
+
+KERNEL_VERSION_SANITY_SKIP = "1"
+
+SRCREV = "af3f4a1caec12845b809fba959e6334ab3b52a40"
+FORK ?= "tekkamanninja"
+BRANCH ?= "allwinner_nezha_d1_devel"
+SRC_URI = "git://github.com/${FORK}/linux.git;protocol=https;branch=${BRANCH} \
+           file://0001-riscv-fix-build-with-binutils-2.38.patch \
+           file://enable-autofs4.cfg \
+           file://enable-cgroups.cfg \
+           file://enable-modules.cfg \
+          "
+
+LINUX_VERSION ?= "5.16.0"
+LINUX_VERSION_EXTENSION:append = "-nezha"
+
+KBUILD_DEFCONFIG = "nezha_defconfig"
+
+COMPATIBLE_MACHINE = "(nezha-allwinner-d1)"
```

Comment:
there is no setting for KERNEL_DEVICETREE variable here. Is it not needed ?

Reply:
The devicetree is provided by platform firmware (u-boot). It is not loaded from disk.